com.unity.multiplayer.mlapi/Runtime/Profiling/PerformanceTickData.cs

```diff
@@ -16,9 +16,12 @@ public void Increment(string fieldName, int count = 1)
 
         public void AddNonDuplicateData(IReadOnlyDictionary<string, int> transportProfilerData)
         {
-            var nonDuplicates = transportProfilerData.Where(entry => !m_TickData.HasData(entry.Key));
-            foreach (var entry in nonDuplicates)
+            foreach (var entry in transportProfilerData)
             {
+                if (m_TickData.HasData(entry.Key))
+                {
+                    continue;
+                }
```

Comment:
I think this fine now. the function name makes it clear enough

Reply:
Yeah, and also +1 as we've been talking about moving away from LINQ in the library